plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -162,7 +173,7 @@ private class DokkaDescriptorVisitor(
         }
     }
 
-    private suspend fun visitClassDescriptor(descriptor: ClassDescriptor, parent: DRIWithPlatformInfo): DClasslike =
+    suspend fun visitClassDescriptor(descriptor: ClassDescriptor, parent: DRIWithPlatformInfo): DClasslike =
```

Comment:
This doesn't seem to leak to public API now (I don't see it in any of the `*.api` files), but I'd prefer `internal` here if possible

Reply:
`DokkaDescriptorVisitor` is private in `DefaultDescriptorToDocumentableTranslator` so `visitClassDescriptor(...)` cannot be used from outside of enclosing class.

Reply:
oh, I see, thanks